Sources/SpeziOnboarding/OnboardingFlow/OnboardingStack.swift

```diff
@@ -49,6 +49,27 @@ import SwiftUI
 ///     }
 /// }
 /// ```
+///
+/// ### Identifying Onboarding Views
+///
+/// Apply the `onboardingIdentifier(_:)` modifier to clearly identify a view in the `OnboardingStack`.
```

Comment:
Can we make this a DocC symbol to properly link to the modifier.

```suggestion
/// Apply the ``SwiftUI/View/onboardingIdentifier(_:)`` modifier to clearly identify a view in the `OnboardingStack`.
```

You might need to verify that this is spelled correctly.